### PR TITLE
write docs

### PR DESCRIPTION
**Pull request by morethanreal**
@cdata PTAL cc @notwaldorf @arthurevans

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,12 +19,12 @@
   <script src="../webcomponentsjs/webcomponents-lite.js"></script>
 
   <link rel="import" href="../polymer/polymer.html">
-  <link rel="import" href="../iron-doc-viewer/iron-doc-viewer.html">
+  <link rel="import" href="../iron-component-page/iron-component-page.html">
 
 </head>
 <body>
 
-  <iron-doc-viewer></iron-doc-viewer>
```
Comment (cdata) [inline index.html]: Woot.

File: iron-input.html [hunk 2]
```diff
@@ -130,8 +150,9 @@
     },
 
     /**
-     * Returns true if the value is valid. The validator provided in `validator` will be used first,
+     * Returns true if `value` is valid. The validator provided in `validator` will be used first,
      * then any constraints.
+     * @return {Boolean} True if the value is valid.
```
Comment (cdata) [inline iron-input.html]: Isn't it supposed to be `boolean`? I can never remember..

**Comment by cdata** [comment]
LGTM :+1:

File: iron-input.html (hunk 2 above)
Comment (morethanreal) [inline iron-input.html]: Yeah, I don't know..is it `boolean` and `number` but `Object`? What about `function`?

Comment (arthurevans) [inline iron-input.html]: Supposedly we take Closure-style type annotations. According to the Closure docs, everything is lowercase (boolean, string, number) except for Object or Array.

https://developers.google.com/closure/compiler/docs/js-for-compiler#types

**Pull request merged**